.github/workflows/test_and_deploy.yml

```diff
@@ -64,34 +28,13 @@ jobs:
         python-version: [3.7, 3.8]
 
     steps:
-      - uses: actions/checkout@v2
-
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v2
+      # Setup pyqt libraries
+      - uses: tlambert03/setup-qt-libs@v1
```

Comment:
Why doesn't this get moved into the composite `test` action? Or is it only used in this repo?

Reply:
The test action is meant to be generic enough to be used without setting up QT libraries, e.g. `cellfinder-core` doesn't need to use setup QT steps.